src/test/java/fi/metatavu/kuntaapi/test/json/NotNullJSONAssertMatcher.java

```diff
@@ -0,0 +1,17 @@
+package fi.metatavu.kuntaapi.test.json;
+
+import org.skyscreamer.jsonassert.ValueMatcher;
+
+/**
+ * Value matcher for JSONAssert that asserts that given value is not null
+ * 
+ * @author Antti Leppä
+ */
+public class NotNullJSONAssertMatcher implements ValueMatcher<Object> {
+
+  @Override
+  public boolean equal(Object o1, Object o2) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either override Object.equals(Object obj), or totally rename the method to prevent any confusion. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1221)